### PR TITLE
add training section

### PR DESCRIPTION
**Pull request by pabloopez**
hi Falco team,

here are the proposed changes to embed the 3 current Falco hands-on labs in the website. One of the labs was released weeks ago and for the other two there's going to be a PR with changes to fix them. So, I believe is a good idea to start moving this at the same time for you to test the proposed changes. 

Thanks for your time! :) 

**What type of PR is this?**

/kind user-interface

/kind content

**Any specific area of the project related to this PR?**

/area training

**What this PR does / why we need it**:

This PR creates a new section in the Falco website to host the hands-on labs. It creates a new section with an introduction to the content with links to one different page for every lab. This way we avoid loading the lab if the user is just viewing the intro. And this way we keep Katacoda infra usage low.

### CODE REVIEW

**Comment by pabloopez** [comment]
Find here a [preview of the changes](https://giphy.com/embed/YIk7RbtUzi8wgFkYRC), not great resolution but it helps to get an idea if you do not want to test it. I checked with `make serve` that all the changes are working fine.

**Review by mstemm — APPROVED** [review]
I'm not sure about the context, but it sounds like @pabloopez talked to other maintainers about adding a training/labs section. @pabloopez, which other maintainers did you talk to? But I did review it using make serve and it looks great.

File: layouts/partials/labs.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<section class="row td-box td-box--4 td-box--gradient td-box--height-auto linkbox">
+  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+    <h2>Lab 1 - Introduction to Falco</h2>
+    <p>General introduction to Falco in single Linux hosts. Learn how to install Falco with Docker, and detect threads
```
Comment (radhikapc) [inline layouts/partials/labs.html:4]: General introduction to Falco running in single Linux hosts. Learn how to install Falco with Docker, and detect threads, such as:

File: layouts/partials/labs.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      like:</p>
+    <ul>
+      <li>Container running interactive shell</li>
+      <li>Unauthorized process</li>
+      <li>Write to non user-data directory</li>
+    </ul>
+    <p>This lab is the best place to start your Runtime Security jurney!</p>
```
Comment (radhikapc) [inline layouts/partials/labs.html:11]: This lab is the best place to start your Runtime Security journey!

File: layouts/partials/labs.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      </a>
+    </div>
+
+  </div>
+
+  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+    <h2>Lab 2 - Falco forensics in K8s</h2>
+    <p>In this lab, you can learn how to deploy Falco in a a running Kubernetes cluster and how to define your first
+      custom rules file.</p>
+    <p>Play the both the intruder & sysadmin roles and detect a Rogue attack</p>
```
Comment (radhikapc) [inline layouts/partials/labs.html:29]: Let us roleplay both the intruder and sysadmin roles and detect a Rogue attack.

File: layouts/partials/labs.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      </a>
+    </div>
+
+  </div>
+</section>
+
+<section class="row td-box td-box--4 td-box--gradient td-box--height-auto linkbox">
+  <div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+    <h2>Lab 3 - Falcosidekick: Thread Response Engine</h2>
+    <p>Falco is not just about Runtime Detection, you can perform automated actions to mitigate threads automatically.
```
Comment (radhikapc) [inline layouts/partials/labs.html:47]: Falco is not just about Runtime Detection.  You can also perform automated actions to mitigate threads automatically.

File: layouts/partials/labs.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    </p>
+    <p>In this lab you will learn how to deploy Falco with Falcosidekick and Kubeless. Stop a running pod when you
```
Comment (radhikapc) [inline layouts/partials/labs.html:49]: In this lab, you will learn how to deploy Falco with Falcosidekick and Kubeless. Stop the running pod when you detect a security thread!

**Review by radhikapc — CHANGES_REQUESTED** [review]
minor suggestions

**Comment by pabloopez** [comment]
Thank @radhikapc for your suggestions, I included a new commit with those changes.

**Review by radhikapc — APPROVED** [review]
LGTM

**Comment by pabloopez** [comment]
> I'm not sure about the context, but it sounds like @pabloopez talked to other maintainers about adding a training/labs section. @pabloopez, which other maintainers did you talk to? But I did review it using make serve and it looks great.

Hi @mstemm, thanks for your time to review this. Your question sounds reasonable to me. Let me add more context:
- An [issue](https://github.com/falcosecurity/evolution/issues/70) was opened in `falco/evolution` to discuss the possibility of unarchiving the existing Falco hands-on labs. 
- Some weeks ago in one of the Falco Community Calls, I introduced the possibility to embed the labs into the Falco website. It was registered in the meeting [agenda](https://hackmd.io/3qYPnZPUQLGKCzR14va_qg?edit) (please, check date: 2021-05-26). 
 
@mstemm  you can find there a list of the assistants, some of them are maintainers of this repository too. If I remember well, I even shared a mockup (slightly different to the final result committed here) to show how we can embed the labs in the Falco website. 

Let me know if more clarification is required to proceed. Thank you.

**Comment by mstemm** [comment]
No other info needed, I read the issue and I'm all caught up now. Thanks!

**Pull request merged**